gradle.properties

```diff
@@ -19,7 +19,7 @@ license=LGPL-3.0-or-later
 # Mod Version
 baseVersion=0.6.25
 # Branch Metadata
-branch=1.21
-tagBranch=1.20
```

Comment:
not quite! the tag branch is the branch we develop features on - that stays at `1.20`.
Additionally, the branch for this is `1.22`